packages/backend-test-utils/api-report.md

```diff
@@ -111,14 +111,7 @@ export namespace mockServices {
   export namespace rootLogger {
     // (undocumented)
     export type Options = {
-      levels:
-        | boolean
-        | {
-            error: boolean;
-            warn: boolean;
-            info: boolean;
-            debug: boolean;
-          };
+      level?: 'none' | 'error' | 'warn' | 'info' | 'debug';
```

Comment:
Hmm this is a breaking change right?

Reply:
Yep but it's an alpha export

Reply:
Yep - just thought about the changeset and calling it out as to what's different?